daemon/src/model/cfd.rs

```diff
@@ -215,15 +217,19 @@ pub enum SettlementKind {
 pub enum CannotAutoRollover {
     #[error("Is too recent to auto-rollover")]
     TooRecent,
-    #[error("Cfd does not have a dlc")]
+    #[error("CFD does not have a DLC")]
     NoDlc,
 }
 
-#[derive(thiserror::Error, Debug, PartialEq)]
-pub enum CannotRollover {
-    #[error("Cfd does not have a dlc")]
+/// Various error cases that can happen during rollover.
+///
+/// This enum is expected to go away once we handle the entire protocol within the actor and not
```

Comment:
I think some of the enum would still remain for defining the error cases that are returned by the rollover function on the `Cfd` - or are we planning to remove this completely then?

Reply:
On that note: It might actually be worth distinguishing `CannotRollover` from `RolloverError` because it's two different concerns. Could potentially map `CannotRollover` into `RolloverError`. (NOt sure it's worth it. Can also keep as is.)